src/commons/repl/ReplInput.tsx

```diff
@@ -40,6 +41,8 @@ export const ReplInput: React.FC<ReplInputProps> = props => {
   const replInput = React.useRef<ReactAce>(null);
   const replInputBottom = React.useRef<HTMLDivElement>(null);
 
+  const isMobileBreakpoint = useMediaQuery(`(max-width: ${Constants.mobileBreakpoint}px)`, true);
```

Comment:
@sayomaki shouldn't call this directly, should be using `useResponsive` instead.